src/main/resources/BridgeServer2.conf

```diff
@@ -78,14 +82,13 @@ local.webservices.url = http://localhost:9000
 route53.zone = ZP0HNVK1V670D
 
 # Synapse Team IDs, used by the BridgeStudyCreator when creating Synapse projects.
-team.bridge.admin = 3388390
-team.bridge.staff = 3388389
+team.bridge.admin = 3336438
+team.bridge.staff = 3336437
```

Comment:
It appears the dev server has all the same accounts and stuff. Why are these changing?

Reply:
These are changing because Dev and Staging used to talk to Synapse Prod. This pull request changes them to talk to Synapse Dev, which means we had to recreate all our resources, and they all have different IDs now.